index-examples.js.ddoc

```diff
@@ -0,0 +1,139 @@
+_=
+$(COMMENT
+Picking the frontpage example after the page has loaded leads to reflows
+and re-rendering. Hence, we pick the example as soon as possible
+Be careful, Ddoc forbids certain patterns, e.g.:
+- using < or >
+- direct assignments (foo = bar) are interpreted as new Ddoc variables
+)
```

Comment:
What prevents adding this as a plain js file under e.g. `js/example_roulette.js`? Can ddoc string-import-mixin other files?